CHANGES.rst

```diff
@@ -5,6 +5,7 @@ Unreleased
 
 -   Remove previously deprecated code.
 -   Pass extra keyword arguments from ``get_or_404`` to ``session.get``. :issue:`1149`
+-   Fix issue with getting per_page value after calling ``next()``. :issue:`1201`
```

Comment:
This needs to be descriptive, "fix issue" doesn't describe what changed. Also needs code formatting around `per_page`. For example:

```
`Pagination.next()` enforces `max_per_page`. :issue:`1201`
```